README.md

```diff
@@ -1310,6 +1311,42 @@ Jwts.parser()
     // ... etc ...
 ```
 
+<a name="json-jackson-custom-parsing"></a>
```

Comment:
Let's have this anchor be `json-jackson-custom-types` to match the section header and content (better SEO)